src/agent/src/netlink.rs

```diff
@@ -255,6 +275,41 @@ impl Handle {
         Ok(list)
     }
 
+    async fn query_rules(&self, ip_version: Option<IpVersion>) -> Result<Vec<packet::RuleMessage>> {
```

Comment:
It seems the default option for `ip_version` is None, why not just get the rule for both ipv4 and ipv6 directly?

Reply:
Thx.  Fixed.